app/controllers/tagged_courses_controller.rb

```diff
@@ -41,9 +42,21 @@ def set_tag
 
   def set_courses_and_presenter
     @courses = Tag.courses_tagged_with(@tag)
-    # This fake Campaign is so that we can reuse campaigns views
-    @campaign = Campaign.new(slug: @tag, title: "Tag: #{@tag}")
     @presenter = CoursesPresenter.new(current_user:, tag: @tag,
                                       courses_list: @courses, page: @page)
   end
+
+  # Loads CoursesUsers records with role 4 and filters by wiki experts, avoiding N+1 queries
+  def load_wiki_experts
+    return @wiki_experts if @wiki_experts # Avoid re-loading if already loaded
```

Comment:
In what situation will @wiki_experts already be loaded?

Reply:
Actually, this wasn't required. I was using it to test something and forgot to remove it.